packages/build/src/plugins/expected_version.ts

```diff
@@ -1,19 +1,25 @@
+import { PackageJson } from 'read-pkg-up'
 import semver from 'semver'
 
+import { FeatureFlags } from '../core/feature_flags.js'
 import { addErrorInfo } from '../error/info.js'
 import { importJsonFile } from '../utils/json.js'
 import { resolvePath } from '../utils/resolve.js'
 
 import { getExpectedVersion } from './compatibility.js'
-import { getPluginsList } from './list.js'
+import { PluginList, getPluginsList } from './list.js'
+import { PluginsOptions } from './node_version.js'
 
-// When using plugins in our official list, those are installed in .netlify/plugins/
-// We ensure that the last version that's been approved is always the one being used.
-// We also ensure that the plugin is our official list.
+/**
+ * When using plugins in our official list, those are installed in .netlify/plugins/
+ * We ensure that the last version that's been approved is always the one being used.
+ * We also ensure that the plugin is our official list.
+ */
```

Comment:
[nit]: changing this comment to jsdoc syntax means the comment should read as docs for the function. As it stands it's a code comment

Reply:
You are right I was just blindly converting all method comments to jsdoc to get the improved intellisense but I was not actually reading them (was a multiline replace file action :D)